demo/src/plugins/dashboard/components/active-edits/selector.js

```diff
@@ -0,0 +1,12 @@
+import getAccessToken from '@triniti/cms/plugins/iam/selectors/getAccessToken';
+import getCollaborationNodes from '@triniti/cms/plugins/raven/selectors/getCollaborationNodes';
+
+export default (state) => {
+  const accessToken = getAccessToken(state);
+  const collaborationNodes = getCollaborationNodes(state).filter((node) => !!node);
```

Comment:
@nanetteCodes i think the !!node filter is already happening in the selector eh?

Reply:
correct.